app/celery/tasks.py

```diff
@@ -239,7 +242,7 @@ def save_sms(self,
         send_notification_to_queue(
             saved_notification,
             service.research_mode,
-            queue=template.queue_to_use()
+            queue=notification.get('queue', template.queue_to_use())
```

Comment:
```suggestion
            queue=notification.get('queue') or template.queue_to_use()
```

Reply:
This might depend on the conversation going on here:
https://github.com/cds-snc/notification-api/pull/1296#discussion_r632715709